index.js

```diff
@@ -97,9 +111,9 @@ else if (program.hub) {
     var fileName = helpers.createOnboardingFileName(program.hub);
     helpers.readFile(fileName, "Please complete onboarding -o").then(data => { 
         var deviceInfo = JSON.parse(data);
-        translatorCli.getProperty(program.hub, deviceInfo, 'HubResURI').then(info => {
+        translatorCli.getProperty(program.hub, deviceInfo, 'get').then(info => {
```

Comment:
>get [](start = 60, length = 3)

Should this be getPlatforms?  They do the same thing today, so it doesn't matter, but we could change get() in the future.  getPlatforms will always return what this call is trying to get.